vsutil/types.py

```diff
@@ -47,4 +56,4 @@ def _resolve_enum(enum: Type[E], value: Any, var_name: str, module: Optional[str
     try:
         return enum(value)
     except ValueError:
-        raise ValueError(f'{var_name} must be in {_readable_enums(enum, module)}.') from None
+        raise ValueError(f"{fn.__name__ + ': ' if fn else ''}{var_name} must be in {_readable_enums(enum)}.") from None
```

Comment:
The function will already be part of the traceback printed on the console (albeit on the second-to-last frame). Do we really need to pass the calling function to mention the function name again?

Reply:
I'd like to keep the standard of `raise Error("func_name: the problem") because this will be used in scripts, which will likely be ran through editors like VSEdit etc., where the person running them might have a hundred other filters and function calls with similar named variables or stuff. The Python tracebacks should make it clear where the error happened but for people new to vs (who likely don't know anything about Python nor what to do with the traceback besides posting it on discord and just saying "help"), printing the error message with the function name that caused the issue is probably a bit more clear IMO.